security/advancedtls/go.mod

```diff
@@ -3,11 +3,10 @@ module google.golang.org/grpc/security/advancedtls
 go 1.13
 
 require (
-	github.com/golang/protobuf v1.3.5 // indirect
 	github.com/google/go-cmp v0.4.0
 	golang.org/x/net v0.0.0-20200602114024-627f9648deb9 // indirect
 	golang.org/x/sys v0.0.0-20200602225109-6fdc65e7d980 // indirect
 	golang.org/x/text v0.3.3 // indirect
-	google.golang.org/genproto v0.0.0-20200513103714-09dca8ec2884 // indirect
-	google.golang.org/grpc v1.29.1
+	google.golang.org/grpc v1.31.0
+	google.golang.org/grpc/examples v0.0.0-20200731180010-8bec2f5d898f
```

Comment:
This looks weird. Can you run `go mod tidy` on your end and see if this dependency still persists?

Reply:
Yeah it still exists after I ran `go mod tidy` :(

Reply:
The tests import `pb "google.golang.org/grpc/examples/helloworld/helloworld"`.

This won't be here if you change it to the test RPCs https://github.com/grpc/grpc-go/tree/master/test/grpc_testing.

Reply:
Ah I see. It's still a bit unclear to me why this dependency didn't show up previously. We had that helloworld dependency a while ago. Is it because something is different in the latest release?
Anyway, I think it makes sense and we can keep it in the current way right now.

Reply:
Didn't run `go mod tidy` when this was added maybe?

Reply:
Ah yeah, that's exactly the case lol. Thanks for the investigation!